File: app/src/main/java/com/example/proyectomovil/Models/Evento.java
package com.example.proyectomovil.Models;

import java.util.ArrayList;
import java.util.List;

public class Evento {
    public int id_evento;
    public String nombre;
    public String descripcion;
    public String imagen;
    public String fecha_inicio;
    public String fecha_fin;
    public String favorito;
    public String calificacion;
    public List<Sitio> sitios = new ArrayList<>();
}

File: ProyectoMovilTour/app/Evento.php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;

class Evento extends Model
{
    protected $table = 'evento';
    protected $primaryKey = 'id_evento';

     protected $fillable = [
    	'nombre',
    	'descripcion',
    	'fecha_inicio',
    	'fecha_fin',
    	'imagen',
    ];
    public function sitios()
	{
		$intersectos = IntersectoEventoSitio::all()->where('id_evento', $this->id_evento);
		$sitios = [];
		foreach ($intersectos as $intersecto) {
			array_push($sitios, $intersecto->sitio);
		}
		return $sitios;
	}
}
